GETTING_STARTED/tutorials.md

```diff
@@ -0,0 +1,46 @@
+# Tutorials
+Here you can find hands-on tutorials and tips for working with Github and the Vagrant box.
+
+## Content
+1. [How to release](#how-to-release-video)
+2. [How to test in vagrant box](#how-to-test-in-vagrant-box-video)
+3. [How to clean up box](#how-to-clean-up-box-video)
+
+## How to release [VIDEO]
```

Comment:
I really like the idea with videos. I would like to offer to create a playlist on youtube

Reply:
It's already in a playlist called *Workflow*: https://www.youtube.com/watch?v=8DbcfoIdbtg&list=PL0k0CoNeXdtLIkPJa5KZqp-02jRDyxEVV&ab_channel=ClaesGill